listen-ap.h

```diff
@@ -36,6 +36,20 @@ bool listenSetWifiConfig(WiFiClient& client) {
         if(res) {
           setWifiConfig(output[0], output[1]);
           Serial.println();
+
+          Serial.println("Sending Data...");
+
+          memset(socketBuff, '\0', MAX_SOCKET_BUFF_SIZE);
+          socketBuff[0] = SUCCESS_RESPONSE_RESULT;
+          socketBuff[1] = WIFI_CONFIG_SUCCESS_RESPONSE_HEADER;
+          const char* ip = WiFi.localIP().toString().c_str();
+          strcpy(socketBuff + 2, ip);
+
+          client.print(socketBuff);
+          Serial.println(socketBuff);
+
+          delay(750);
+          client.stop();
```

Comment:
This change looks strange
Pls ensure that last version of master is merged

Reply:
This part is merged. but maybe incorrect. 
I`ll try to fix this. 
If I fail - maybe on meeting we will fix this issue?

Reply:
will fix that myself